felix/bpf-gpl/nat_types.h

```diff
@@ -121,4 +122,23 @@ struct vxlanhdr {
 	__be32 flags;
 	__be32 vni;
 };
-#endif /*  __CALI_NAT_TYPES_H__ */
+
+struct calico_ch_key {
+	ipv46_addr_t vip;
+	__u16 port;
+	__u8 proto;
+	__u8 pad;
+	__u32 ordinal; // should always be a value of [0..M], where M is a very large prime number. -Alex
+};
+
+// 65537 * 100 gives a large lookup table for up to 100 services. -Alex
```

Comment:
Shouldn't it match `BPFMapSizeConsistentHash           int               `config:"int;1009500;non-zero"`` ?